[PAN-1594] useReanalysisCorrection is not only controlled by BuiltInWindParameters class

ivan-fursa-aurora opened the pull request:
Jira: https://auroraenergy.atlassian.net/browse/PAN-1594

Please see testing and context in the Jira ticket

danielth-uk commented on aurora/amun/client/parameters.py:
```diff
@@ -164,7 +161,7 @@ class BuiltInWindParameters(FlowParameters):
         self,
         windType: WindType,
-        useReanalysisCorrection: bool = None,
+        useReanalysisCorrection: bool = False,
```
Please update line 156 (I cant comment directly on it) The documentation of the default value is wrong.

```diff
 useReanalysisCorrection (bool, optional): Should Regional Reanalysis Correction be enabled.
-            If true then a location specific *reanalysisScaleCorrectionDelta* is used. Defaults to None.
  useReanalysisCorrection (bool, optional): Should Regional Reanalysis Correction be enabled.
+            If true then a location specific *reanalysisScaleCorrectionDelta* is used. Defaults to False
```

danielth-uk commented on docusaurus-site/docs/Reference/parameters.md:
This file on line 155 also needs to be changed to reflect the new default value

danielth-uk commented on tests/test_raw_generation_upload.py:
```diff
@@ -49,7 +48,7 @@ def test_load_factor_to_valuation(amun_session, snapshot):
         "hubHeight": hub_height,
         "obstacleHeight": obstacle_height,
-        "useReanalysisCorrection": use_reanalysis_correction,
+        "useReanalysisCorrection": False,
```
Why has this been taken away from using the variable?

e.g Why has it gone from this? to this?

```diff
- use_reanalysis_correction = False
...
        "obstacleHeight": obstacle_height,
-        "useReanalysisCorrection": use_reanalysis_correction,
+        "useReanalysisCorrection": False,

```

ivan-fursa-aurora replied:
That's because we no longer pass it to the load factor call. However I do agree that this change is not necessary. It's best I leave things as they were in favour of reusability of a variable

danielth-uk reviewed (changes requested):
The documentation of `BuiltInWindParameters` needs to be updated to reflect changes. Also if there is a new version of the documentation, could you also please update the changelog and version number (since it is technically new documentation although not a big issue if you don't think its important)

ivan-fursa-aurora commented:
Thanks, that's a really useful review!

Addressed all your comments.

Some screenshots:

![image](https://github.com/AuroraEnergyResearch/aurora-amun-python-sdk/assets/130995610/72545015-b63c-493f-9f5f-eb70f6d2f3c0)

![image](https://github.com/AuroraEnergyResearch/aurora-amun-python-sdk/assets/130995610/1e9bf368-10f1-41c1-90ec-e64adbca93d2)

![image](https://github.com/AuroraEnergyResearch/aurora-amun-python-sdk/assets/130995610/7230f7cd-fa11-4ac1-a01f-a95a69a864f0)

![image](https://github.com/AuroraEnergyResearch/aurora-amun-python-sdk/assets/130995610/abef47e2-722f-468e-993a-0522689a6313)

danielth-uk reviewed (approved):
👍
